### PR TITLE
Fix math rendering in Markdown

### PR DESCRIPTION
**Pull request by tchamelot**
resolves avisi-cloud/structurizr-site-generatr#443

### CODE REVIEW

File: src/main/resources/assets/js/katex-render.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+(function () {
```
Comment (jp7677) [inline src/main/resources/assets/js/katex-render.js:1]: Could you please add a comment with the source of that script?

**Review by jp7677 — CHANGES_REQUESTED** [review]
Thanks a lot for diving into this.
Just one small nit, otherwise LGTM!

File: src/main/resources/assets/js/katex-render.js (hunk 1 above)
Comment (tchamelot) [reply src/main/resources/assets/js/katex-render.js:1]: I amended my commit to include a comment with a link to flexmark wiki where the script come from

**Review by jp7677 — APPROVED** [review]

**Comment by jp7677** [comment]
Very cool, thanks for this PR!

**Pull request merged**